examples/seq2seq/test_seq2seq_examples.py

```diff
@@ -311,6 +312,58 @@ def test_run_eval(model):
         assert Path(output_file_name).exists()
         os.remove(Path(output_file_name))
 
+@slow
+@pytest.mark.parametrize("model", [pytest.param(T5_TINY), pytest.param(BART_TINY), pytest.param(MBART_TINY)])
```

Comment:
```suggestion
@pytest.mark.parametrize("model", [pytest.param(T5_TINY)])
```

Reply:
so no, since one model will only test one feature - we need to test both (trans/summ), but I reduced it to 2 models - each testing one feature.